src/sui/themes/rtd-site/elements/button.overrides

```diff
@@ -0,0 +1,113 @@
+.ui.button {
+  vertical-align: middle;// Required for .button to be aligned with .button.animated
+}
+
+
+.ui.basic.buttons form > .button {
+  &:last-child {
+    border-top-right-radius: 0rem;
+    border-bottom-right-radius: 0rem;
+  }
+
+  &:hover {
+    box-shadow: none;
+  }
+
+  border-left: 0rem;
+}
+
+/*******************************
+     Gradient Variants (new)
```

Comment:
First time I'm using it really... just because the rest of the SUI code was using it. And I like to keep these details looking consistent. Guess that's why I'm frontend :upside_down_face:

Reply:
Feel free to go as minimal as you like! I'll show you some fancy features we can use to integrate with RTD on source code soon, then we can get really weird with documenting our code.

My editor seems to hate this formatting for some reason :shrug:

Reply:
Gotcha